pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -387,6 +387,7 @@ func (f *framework) ApplyScoreWeights(pc *PluginContext, pod *v1.Pod, scores Plu
 		if !ok {
 			err := fmt.Errorf("score plugin %v has no corresponding scores in the PluginToNodeScoreMap", pl.Name())
 			errCh.SendErrorWithCancel(err, cancel)
+			return
```

Comment:
Thanks for the catch. I believe there are other occurrences in this or related files. Can you please change them all?

/assign @ahg-g 
for a thorough check.

Reply:
sg, please do the same throughout the file as @Huang-Wei suggested.

There are a couple of cases in https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/algorithm/priorities/interpod_affinity.go as well